index.html

```diff
@@ -67,18 +68,25 @@
       <!-- Example row of columns -->
       <div class="row content">
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
+          <h2>About Us</h2>
+          <div class="justify-text">
+          <p>LinuxChix India is a regional chapter in India of LinuxChix, a global community for women who like Linux
+          and for anyone who wants to support <strong>women in computing</strong>. We are an international group of Free Software
+          users and developers, founded in 1999 with the aim of supporting women in Linux. It is not only for women.</p>
+        </div>
           <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
         </div>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec id elit non mi porta gravida at eget metus. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus. Etiam porta sem malesuada magna mollis euismod. Donec sed odio dui. </p>
-          <p><a class="btn btn-default" href="#" role="button">View details &raquo;</a></p>
-       </div>
+          <h2>Our Motto</h2>
+          <p style="font-size:50px;color:#F5C8D5;">Be Polite.</br>Be Helpful. </p>
+        </div>
         <div class="col-md-4">
-          <h2>Heading</h2>
-          <p>Donec sed odio dui. Cras justo odio, dapibus ac facilisis in, egestas eget quam. Vestibulum id ligula porta felis euismod semper. Fusce dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh, ut fermentum massa justo sit amet risus.</p>
+          <h2>Our Aim and Vision</h2>
+            <div class="justify-text">
+          <p style="text-align:justify">We aim to bring together women around India involved in various FOSS projects, enhance women participation
```

Comment:
`<p>` tag is a child to div with class `justify-text`. Please ensure it is properly indented by 2 spaces with respect to parent.